### DISCUSSION TITLE
Селект-дерево

### DISCUSSION THREAD

**Discussion by Makhmut2445** [totumonline/totum-ru-issues-and-discussions#104] [Как это сделать?] [answered]
Можно пример полноценного кода селект и его результата? 
С разъяснениями что к чему.

**Comment by ttm-alexeypolunin** [comment]
Будет, но не прям завтра

**Comment by ttm-alexeypolunin** [comment, marked as answer]
### [Временное видео на YouTube —>](https://youtu.be/TBHHT-AZHvs)
